### PR TITLE
DDCNLS-691 change how the CSRF token is referenced within the applica…

### PR DESCRIPTION
**Pull request by MatthewJohnsonDDCN**
…tion.conf

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -47,7 +47,28 @@ include "validation-config/SAYE/ers-saye-exercised-validation.conf"
...
+  http {
+    router = prod.Routes
+    requestHandler="play.api.http.GlobalSettingsHttpRequestHandler"
+  }
+  modules.enabled+="com.kenshoo.play.metrics.PlayModule"
+  filters {
+    csrf {
+      cookie.secure=true
+      token.sign=false
+      token.name=ERS_CSRF
```
Comment (mplanchant) [inline conf/application.conf]: I think token.name needs to be unique. Maybe "ERS_CHECK_CSRF"?

Comment (mplanchant) [inline conf/application.conf]: I think it needs to be in quotes too. See [https://www.playframework.com/documentation/2.5.x/resources/confs/filters-helpers/reference.conf.](https://www.playframework.com/documentation/2.5.x/resources/confs/filters-helpers/reference.conf).

**Comment by mplanchant** [comment]
@rambabu-posa are you happy with non-unique `token.name`?

**Comment by rambabu-posa** [comment]
Please use unique CSRF token name. As ERS-Returns-Frontend is already using ERS_CSRF, please use something different like "ERS_CHECK_CSRF" or  "ERS_CHECKING_CSRF" etc.

**Comment by rambabu-posa** [comment]
Yes please.

**Comment by rambabu-posa** [comment]
You can comment or remove these separate entries

play.modules.enabled
play.http.requestHandler

Because Play CSRF block is covering them.

**Pull request merged**